### ISSUE TITLE
Add accessibility info to website

### ISSUE THREAD

**Issue by karawoo** [karawoo/2015-04-27-wsu#1]
We should add information about accessibility to the website, something along the lines of https://github.com/swcarpentry/workshop-template/pull/160. 

Unfortunately the building we will be in is highly inaccessible - no elevators, accessible restrooms, or accessible parking. I'm going to look into what other kinds of accessibility accommodations we can make available. I'm happy to give rides to/from the front door of the building, though this probably will not help people who use wheelchairs or otherwise can't climb stairs. I _think_ it would be possible to create lactation facilities -- IIRC the room we'll be in has a sink and fridge and some smaller adjoining rooms that could be used. I'm trying to get access to the room to double check this. Large print handouts should be no problem. I'll contact the Access Center at WSU and our hosts to see what else we might be able to provide and then send a PR adding all the info to the website.
